docs/setup-guide/cluster_setup.md

```diff
@@ -0,0 +1,422 @@
+#	Edge Cluster Multi-Layer Setup and Configuration
+
+## Abstract
+The purpose of this document is to how to setup and configuration Cloud core and Edge core , and describe the each step to create virtual machine, setup the port number, install kubernets, GoLang, and so on. Running Cloud core and Edge core and deployed mission and task to Edge node. Improve the Edge computing. This Cloud and Edge design is derived from cloud end to edge end for Edge System Functional Description and the Setup Requirements Specification. The intended user of this program is the edge computing user. 
+1. Virtual Machine Setup (create Cloud core  and Edge core virutal machine, and setup port),
+2. Fornax Installation and Configuration (Install all the kubernetes compnents: kubectl, kubadm, bubelet),
+3. GoLang Installlation and Configuration (Install all GoLang component and load the Fornax source code), 
+4. Generate Machine Security certification, and deployed to virtual machine, 
+5. Install CRD file in Cloud core. 
+6. Run cloud-core and edge-core and deployed mission and verify the mission.
+
+## 1.1. Virtual Machine Setup and Configuration (We use AWS for example)
+-	Ubuntu 18.04, one for cloud-core, two for edge-core.
+-	Open the port of 10000 and 10002 in the security group of the cloud-core machine and edge-core machine
+- 	Go to doc and follow up instruction to setup: Virtual Machine Setup and Configuration <a href="vm_setup.md" target="_blank"> Virtual Machine Setup and Configuration </a>
+- 	After done, you can continue to 1.2.
+
+## 1.2.	Install Kubernetes Tools to Cloud core and Edge core
+-	Install kubernetes tools to virtual machine.(Make sure install version is: 1.21.1-00).
+- 	<a href="https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#installing-kubeadm-kubelet-and-kubectl" target="_blank">Kubernetes Tools Doc</a>
```

Comment:
change to this

Install [kubernetes tools]( https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#installing-kubeadm-kubelet-and-kubectl) to virtual machine.(Make sure install version is: 1.21.1-00).

Use []() format to insert url instead of <a></a>